u-boot/rpi_arm64/0001-nvme-support.patch

```diff
@@ -0,0 +1,407 @@
+# This patch is based on https://github.com/home-assistant/operating-system/commits/755239536812b60f7ebe1c4c679b39597c3f560d/buildroot-external/board/raspberrypi/patches
```

Comment:
let's use the individual patches as is from buildroot-external, as opposed to a single file, makes it easier to remove patches when things gets upstreamed

Reply:
I had to modify some patches to work with the 2022.07. release & therefore bundled them. I'll update this PR, giving u-boot 2023.01 (released yesterday) a spin to see if it works for the remaining platforms (jetson&rockpi)

Reply:
Didn't work out for the jetson

Reply:
yeh, the jetson is one thing preventing us from upgrading right now :cry:

Reply:
hmm I seem to have fixed the patches for u-boot 2023.01, might make your work easier

Reply:
first need to test the build :crossed_fingers:

Reply:
Let me know!

I took the 2023.01 patches and backported them to 2022.07

I know took the 2022.01. patches and adapted them for 2022.07 but it's not working - I'll spend some more time on that this evening or on the weekend

Reply:
It did!